qutip/core/operators.py

```diff
@@ -334,10 +345,11 @@ def sigmam():
      [ 1.  0.]]
 
     """
-    return _SIGMAM.copy()
+    dtype = dtype or settings.core["default_dtype"] or _data.CSR
```

Comment:
Perhaps we should have a function for this line? E.g. Perhaps `.to` could take a `default_dtype` parameter or similar. Depends whether it would help in enough cases.

Reply:
From discussion with @Ericgig: Happy to leave this as is pending further conversations around better default handling (e.g. having both a dense and a sparse default).